### ISSUE TITLE
Consulta sobre estructura del archivo

### ISSUE THREAD

**Issue by Ftrivelloni** [PI-ITBA/2024_01#411]
Bunas tardes, tengo una pequeña duda acerca de esta parte del enunciado

![image](https://github.com/PI-ITBA/2024_01/assets/166187838/f89682d3-4188-494b-95b9-485ab2652837)

Lo que tengo entendido de esto es que en todo archivo que le llegue al main. Lo que nunca cambia es la forma/orden en la que se presenta la informacion. Para el caso de **ticketsCHI.csv**  por ejemplo seria:

**issueDate;plateRedacted;unitDescription;infractionCode;fineLevel1Amount**

pero lo que podria cambiar de archivo a archivo seria, por ejemplo, el formato de la patente. Esta bien mi interpretacion?

De antemano muchas gracias!

**Comment by marcelogarberoglio** [comment]
El formato de cada dato es el que se especifica en el enunciado para cada campo. En el caso de las patentes dice "cadena de hasta 10 caracteres inclusive"

**Comment by andresgarbarz** [comment]
> El formato de cada dato es el que se especifica en el enunciado para cada campo. En el caso de las patentes dice "cadena de hasta 10 caracteres inclusive"

Yo aún tengo una duda respecto a lo que se consultó en este issue. ¿Podemos asumir que en el CSV que nos envía el usuario se respeta el orden de columnas dado en los ejemplos? Es decir, en el caso del archivo infractionsNYC.csv publicado por ejemplo, los datos figuran en el orden id;description.
Entonces, ¿Puedo asumir que en cada línea de información del archivo de infracciones que me pase el usuario para el ejecutable de NYC, el primer elemento será un id y el segundo una descripción?

Gracias de antemano.

**Comment by fmeola** [comment]
Sí, pueden asumir que el orden de las columnas se respeta para los 4 CSVs
